src/_pytest/runner.py

```diff
@@ -340,9 +340,9 @@ def pytest_runtest_makereport(item: Item, call: CallInfo[None]) -> TestReport:
 
 def pytest_make_collect_report(collector: Collector) -> CollectReport:
     call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
-    longrepr = None  # type: Union[None, Tuple[str, int, str], str, TerminalRepr]
+    longrepr: Union[None, Tuple[str, int, str], str, TerminalRepr] = None
     if not call.excinfo:
-        outcome = "passed"  # type: Literal["passed", "skipped", "failed"]
+        outcome: Literal["passed", "skipped", "failed"] = "passed"
```

Comment:
unfortunately there's a catch with `Literal` -- you can't use it globally like

```python
OutCome = Literal[...]
```

because `Literal` is not available (needs guarding by `TYPE_CHECKING`)

so then you quote it:

```python
OutCome = "Literal[...]"
```

but now it's not a type annotation, it's just a string literal

so to make it work you can hack around it and wrap it in a trivial `Union` but then that's harder to explain that just copy paste imo

```python
OutCome = Union["Literal[...]"]
```

Reply:
There's the [`typing_extensions` PyPI module](https://pypi.org/project/typing-extensions/) which backports `Literal` and other newer additions - but not sure if it's worth adding it.